src/public/templates/tv.html

```diff
@@ -50,357 +52,355 @@
 
 <div class="tv-guide-container none">
 
-<a href="javascript:void(0)" class="grid-settings"></a>
+    <a href="javascript:void(0)" class="grid-settings"></a>
 
-<div class="now-pointer" style="left: 140px;">
-    <span>Now</span>
-</div>
-
-<div class="hours">
-    <span>Today</span>
-    <span>12:00AM</span>
-    <span>12:30AM</span>
-    <span>1:00AM</span>
-    <span>1:30AM</span>
-    <span>2:00AM</span>
-    <span>2:30AM</span>
-    <span>3:00AM</span>
-    <span>3:30AM</span>
-    <span>4:00AM</span>
-    <span>4:30AM</span>
-    <span>5:00AM</span>
-    <span>5:30AM</span>
-</div>
-
-<div class="channels">
-<a href="javascript:void(0)">MTVHD 230</a>
-<a href="javascript:void(0)">MTVSD 130</a>
-<a href="javascript:void(0)">MTVSD 130</a>
-<a href="javascript:void(0)">MTVSD 130</a>
-<a href="javascript:void(0)">MTVHD 230</a>
-<a href="javascript:void(0)">MTVSD 130</a>
-<a href="javascript:void(0)">MTVSD 130</a>
-<a href="javascript:void(0)">MTVSD 130</a>
-<a href="javascript:void(0)">MTVHD 230</a>
-<a href="javascript:void(0)">MTVSD 130</a>
-<a href="javascript:void(0)">MTVSD 130</a>
-<a href="javascript:void(0)">MTVSD 130</a>
-</div>
+    <div class="now-pointer" style="left: 140px;">
+        <span>Now</span>
+    </div>
 
-<div class="guide-content container">
+    <div class="hours">

```

Comment:
note for the future: not all countries use 12h time

Reply:
i didnt add this